src/devices/quhwa.c

```diff
@@ -0,0 +1,53 @@
+/* Quhwa
+ *
+ * Tested devices:
+ * QH-C-CE-3V (which should be compatible with QH-832AC)
+ *
+ * Copyright (C) 2016 Ask Jakobsen
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+#include "rtl_433.h"
+#include "pulse_demod.h"
+
+
+static int quhwa_callback(bitbuffer_t *bitbuffer) {
+	bitrow_t *bb = bitbuffer->bb;
+	uint8_t *b = bb[0];
+	b[0] = ~b[0];
+	b[1] = ~b[1];
+	b[2] = ~b[2];
+	
+	unsigned bits = bitbuffer->bits_per_row[0];
+
+	if ((bits == 18) &&
+	    (b[2]==0xFF) &&
+	    (b[1] & 1) && (b[1] & 2)) // Last two bits are one
+	  {
+	    uint32_t ID = (b[0] << 8) | b[1];
+	    
+	    fprintf(stdout, "Quhwa\n");
+	    fprintf(stdout, "ID 14bit = 0x%04X\n", ID);
```

Comment:
You have to use data_make() and friends. Look in waveman.c to see how it is done.